app/src/main/java/com/isc/hermes/model/Utils/IncidentsUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.isc.hermes.model.Utils;
+
+import org.bson.types.ObjectId;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+/**
+ * Utility class for handling incidents-related operations.
+ */
+public class IncidentsUtils {
```

Comment:
One suggestion that you could do is create a private constructor to prevent this class from being instantiated.

Reply:
Since I changed the class method to be singleton, this observation becomes invalid.